src/gui/sdlmain.cpp

```diff
@@ -1254,20 +1288,43 @@ SDL_Rect GFX_GetSDLSurfaceSubwindowDims(uint16_t width, uint16_t height)
 	return rect;
 }
 
+// kvas kompot pelmeni shashlyki kotlety bliny belyashi
```

Comment:
Really? 😁

Reply:
Stumbled upon a line in DoomRL sources:
```// pieczarki oliwki szynka kielbasa peperoni motzarella //```
It got me _so_ inspired!..

Reply:
Ah! thanks for sharing that inside joke. 
I suspected something more.. but didn't get the reference (and indeed, you never disappoint!)